app/models/user.rb

```diff
@@ -0,0 +1,8 @@
+class User < ActiveRecord::Base
+  has_many :surveys
+  has_many :questions
+  has_many :choices
+  has_many :answers
+  validates :username, presence: true
```

Comment:
Probably put some password validations here, too, more than the default of it must be present (which you get with has_secure_password)